src/DataStructures/Tensor/Expressions/AddSubtract.hpp

```diff
@@ -82,30 +82,25 @@ struct AddSub<T1, T2, ArgsList1<Args1...>, ArgsList2<Args2...>, Sign>
 
   AddSub(T1 t1, T2 t2) : t1_(std::move(t1)), t2_(std::move(t2)) {}
 
-  template <typename... LhsIndices, typename T, int U = Sign,
-            Requires<U == 1> = nullptr>
+  template <typename... LhsIndices, typename T, int U = Sign>
   SPECTRE_ALWAYS_INLINE type
   get(const std::array<T, num_tensor_indices>& tensor_index) const {
-    return t1_.template get<LhsIndices...>(tensor_index) +
-           t2_.template get<LhsIndices...>(tensor_index);
+    if constexpr (U == 1) {
+      return t1_.template get<LhsIndices...>(tensor_index) +
+             t2_.template get<LhsIndices...>(tensor_index);
+    } else if (U == -1) {
+      return t1_.template get<LhsIndices...>(tensor_index) -
+             t2_.template get<LhsIndices...>(tensor_index);
+    }
   }
 
-  template <typename... LhsIndices, typename T, int U = Sign,
-            Requires<U == -1> = nullptr>
-  SPECTRE_ALWAYS_INLINE type
-  get(const std::array<T, num_tensor_indices>& tensor_index) const {
-    return t1_.template get<LhsIndices...>(tensor_index) -
-           t2_.template get<LhsIndices...>(tensor_index);
-  }
-
-  template <int U = Sign, Requires<U == 1> = nullptr>
-  SPECTRE_ALWAYS_INLINE typename T1::type operator[](size_t i) const {
-    return t1_[i] + t2_[i];
-  }
-
-  template <int U = Sign, Requires<U == -1> = nullptr>
+  template <int U = Sign>
```

Comment:
Also don't need the `U = Sign` here

Reply:
Done! I ended up removing the empty `template <>` that would remain because I didn't know if there was any practical reason to keep it. Let me know if I should leave it was a templated function with no template parameters, instead.  Also changed the `else if` to `else if constexpr`.

Reply:
Sounds good!